### PR TITLE
Hifi improvised topic

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

This PR is basically solving all margin/padding related issues.

Issues Document Link: https://docs.google.com/document/d/1VJQa91i5Eq5xCe7KKZoT7d8HAWOjE50hvhs54F8_e7s/edit?usp=sharing

Mock Link:https://xd.adobe.com/spec/e2239cf4-9cde-4c08-5296-25316c1f0a14-9412/screen/19cfbacf-854c-4c7d-8691-3b3d117e1866/TP-Overview-

## Screenshots
Screenshot_1574326010
Screenshot_1574328641
Screenshot_1574328645
Screenshot_1574330171
Screenshot_1574330176
Screenshot_1574332364
Screenshot_1574333043

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout/topic_train_fragment.xml [hunk 1]
```diff
@@ -15,72 +15,69 @@
...
-          android:textColor="@color/oppiaSecondaryText"
-          android:textSize="16sp" />
+      <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/skill_recycler_view"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="40dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginEnd="20dp"
+        app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager" />
```
Comment (veena14cs) [inline app/src/main/res/layout/topic_train_fragment.xml:66]: There is issue over here if we have more items in the list. I think we should remove padding bottom from LinearLayout. And As Recyclerview is inside scrollview, scrolling is giving problem when list is big. Please check

untitled

**Review by veena14cs — CHANGES_REQUESTED** [review]
Please address one issue with scrolling in train tab.

**Review by nikitamarysolomanpvt — APPROVED** [review]
LGTM

**Comment by rt4914** [comment]
> Please address one issue with scrolling in train tab.

I think we will need to change the implementation of recyclerview in this, we will have to create this in a similar that we created other recyclerviews like HomeFragment and StateFragment.

File: app/src/main/res/layout/topic_train_fragment.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/layout/topic_train_fragment.xml:66]: I think as this PR is mostly on on Hifi, I will be creating a separate PR for this issue.
I have filed a issue to solve this. #468 
So I believe this PR can be merged and this issue will get solved in #468

**Comment by veena14cs** [comment]
> > Please address one issue with scrolling in train tab.
> 
> I think we will need to change the implementation of recyclerview in this, we will have to create this in a similar that we created other recyclerviews like HomeFragment and StateFragment.

Ok rest all things looks good.

**Review by veena14cs — APPROVED** [review]

**Pull request merged**